examples/alphaTestExamples/MultipleServices.md

```diff
@@ -58,7 +58,9 @@ the namespace and inventory id used by apply to create inventory objects.
 <!-- @createInventoryTemplate @testE2EAgainstLatestRelease-->
 ```
 kapply init $BASE/mysql
+sed -i 's/name: inventory/name: inventory-mysql/g' $BASE/mysql/inventory-template.yaml
```

Comment:
We need to have an e2e test where the initial state has multiple inventory objects in the cluster, and apply consolidated them to single object and preview let it as is.